### ISSUE TITLE
[FEATURE] Scaffold “Rounded” rotations.

### ISSUE THREAD

**Issue by liquidsquid1** [CCBlueX/LiquidBounce#3920]
### LiquidBounce Branch

Nextgen

### Describe your feature request.

There should be a rotation mode for scaffold that will try to get it to rotate in the direction closest to the 90th degree. For example, it will first try reverse yaw, then reverse yaw with a 90 degree offset, then -90, etc. If it cannot aim in any of the 90 degree offsets, it’ll try again with 45 degrees and so on. This will make the rotations for the scaffold a bit more bearable on anticheats like NCP when towering, and on anticheats like grim because of the fact that your movement should be corrected perfectly to where you are moving.

### Additional context

TL;DR: New scaffold rotation mode where it tries all 90 degree angles for rotation, then 45 degree and so on.

**Comment by larryngton2** [comment]
my brain isnt braining

**Comment by superblaubeere27** [comment]
ok, what speaks against Nearest or Stabilized aiming in this situation?

**Comment by liquidsquid1** [comment]
> ok, what speaks against Nearest or Stabilized aiming in this situation?

I have noticed that the scaffold has difficulty in towering on some anticheats because of the rotations alone, and these rotations are alot simpler and should have an easier experience bypassing on anticheats like nocheatplus

**Comment by larryngton2** [comment]
> > ok, what speaks against Nearest or Stabilized aiming in this situation?
> 
> I have noticed that the scaffold has difficulty in towering on some anticheats because of the rotations alone, and these rotations are alot simpler and should have an easier experience bypassing on anticheats like nocheatplus

like godbridge rots with predictions?

**Comment by liquidsquid1** [comment]
> > > ok, what speaks against Nearest or Stabilized aiming in this situation?
> > 
> > 
> > I have noticed that the scaffold has difficulty in towering on some anticheats because of the rotations alone, and these rotations are alot simpler and should have an easier experience bypassing on anticheats like nocheatplus
> 
> like godbridge rots with predictions?

like nearest rotation but it rounds it to the nearest 90 degree first, then 45 degree if it can't find it, then 23 degree, etc.

**Comment by superblaubeere27** [comment]
> > ok, what speaks against Nearest or Stabilized aiming in this situation?
> 
> I have noticed that the scaffold has difficulty in towering on some anticheats because of the rotations alone, and these rotations are alot simpler and should have an easier experience bypassing on anticheats like nocheatplus

maybe we should just fix the other rotation modes then

**Comment by larryngton2** [comment]
> > > ok, what speaks against Nearest or Stabilized aiming in this situation?
> > 
> > 
> > I have noticed that the scaffold has difficulty in towering on some anticheats because of the rotations alone, and these rotations are alot simpler and should have an easier experience bypassing on anticheats like nocheatplus
> 
> maybe we should just fix the other rotation modes then

yeah

so when jumping try to rotate straight?

**Comment by superblaubeere27** [comment]
@larryngton2 so there is an issue with the rotation modes and jumping?

**Comment by larryngton2** [comment]
> @larryngton2 so there is an issue with the rotation modes and jumping?

not sure, I do not fucking know what he wants
